portfolio/src/main/webapp/style.css

```diff
@@ -14,101 +16,310 @@
     margin-right: auto;
     width: 50%;
 }
-body {
-    background-image: url('https://img.freepik.com/free-photo/white-concrete-wall_53876-92803.jpg');
-}
-h1 {text-align: center;}
+
+/* New portfolio styling */
+
 *{
-padding:0;
-margin:0;
-text-decoration: none;
-list-style: none;
-box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+	font-family: 'Josefin Sans', sans-serif;
+	box-sizing: border-box;
 }
-
-body{
-    font-family: montserrat;
+.hero{
+	height: 100vh;
+	width: 100%;
+	background-image: url(/images/background.jpg);
+	background-size: cover;
+	background-position: center;
 }
 nav{
-    background: #0082e6;
-    height: 80px;
-    width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding-top: 45px;
+	padding-left: 8%;
+	padding-right: 8%;
 }
-label.logo{
-    color: white;
-    font-size: 35px;
-    line-height: 80px;
-    padding: 0 100px;
-    font-weight: bold;
+.logo{
+	color: white;
+	font-size: 35px;
+	letter-spacing: 1px;
+	cursor: pointer;
 }
-nav ul{
-    float: right;
-    margin-right: 20px;
+span{
+	color: #f9004d;
 }
 nav ul li{
-    display: inline-block;
-    line-height: 80px;
-    margin: 0 5px;
+	list-style-type: none;
+	display: inline-block;
+	padding: 10px  25px;
 }
 nav ul li a{
-    color: white;
-    font-size: 17px;
-    padding: 7px 13px;
-    border-radius: 3px;
-    text-transform: uppercase;
-}
-a.active,a:hover{
-    background: #1b9bff;
-    transition: .5s;
-}
-.checkbtn{
-    font-size: 30px;
-    color: white;
-    float: right;
-    line-height: 80px;
-    margin-right: 40px;
-    cursor: pointer;
-    display: none;
-}
-#check{
-    display: none;
-}
-@media (max-width: 952px){
-    label.logo{
-        font-size: 30px;
-        padding-left: 50px;
-    }
-    nav ul li a{
-        font-size: 16px;
-    }
-}
-@media (max-width: 858px){
-    .checkbtn{
-        display: block;
-    }
-    ul{
-        position: fixed;
-        width: 100%;
-        height: 100vh;
-        background: #2c3e50;
-        top: 80px;
-        left: -100%;
-        text-align: center;
-        transition: all .5s;
-    }
-    nav ul li{
-        display: block;
-        margin: 50px 0;
-        line-height: 30px;
-    }
-    nav ul li a{
-        font-size: 20px;
-    }
-    a:hover,a.active{
-        background: none;
-        color: #0082e6;
-    }
-    #check:checked ~ ul{
-        left: 0;
-    }
-}
\ No newline at end of file
+	color: white;
+	text-decoration: none;
+	font-weight: bold;
+	text-transform: capitalize;
+}
+nav ul li a:hover{
+	color: #f9004d;
+	transition: .4s;
```

Comment:
[4.1.10](https://google.github.io/styleguide/htmlcssguide.html#Leading_0s): Always include leading “0”s in values.